packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/gestures.dart' show DragStartBehavior;
+import 'package:flutter/material.dart' show debugCheckHasMediaQuery;
```

Comment:
We don't want to import material into widgets. If the debugCheckHasMediaQuery is implemented there we should move it to widgets, and then re-export it from material so we don't break anything